proftpd/init.sls

```diff
@@ -19,7 +19,13 @@ proftpd:
         - makedirs: True
         - template: jinja
         - context:
-          Modules: {{ proftpd.Modules }}
+          Modules: {{ proftpd.Modules.enabled }}
```

Comment:
`context` (and `defaults`) require another level of indentation:

```suggestion
        - context:
            Modules: {{ proftpd.Modules.enabled }}
```

Note, this may need to be reset to `{{ proftpd.Modules }}`, if the plan is to do the enabling/disabling in the template instead.